magento/magento2
# Add compare list link to success message after adding a product

### Description
If I add a product to the compare list, I want to go the compare list directly after that in many cases without having to search for a link. Thus, the link has been added to the success message now.

### Manual testing scenarios
1. Go to a product view page
2. Click "Add to compare"
3. Check if the success message contains a link to the compare list:
![grafik](https://user-images.githubusercontent.com/662059/36728546-6012f1d4-1bc1-11e8-8554-1aebae427ede.png)

### Contribution checklist
